addons/binding/org.openhab.binding.digiplex/src/main/java/org/openhab/binding/digiplex/communication/AbstractResponse.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.digiplex.communication;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Abstract response
```

Comment:
Can you add some javadoc, that explains the concepts and when to use this abstract class?